src/configs/base_keyspecs.c4m

```diff
@@ -1640,6 +1640,22 @@ Platform passed when performing `docker build`, if any.
 """
 }
 
+keyspec DOCKERFILE_VCTL_REL_PATH {
```

Comment:
@viega opinions on abbreviations in the keys? we use some like `OP`, etc but for the most part is spelled out

thoughts on:

* `DOCKERFILE_VCTL_RELATIVE_PATH`
* or simply `DOCKERFILE_VCTL_PATH` since the `VCTL` already means a repo so the path must be relative within git

Reply:
@viega lmk! everything else is squared away.